### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by Fabien-Chouteau**

### CODE REVIEW

**Comment by simonjwright** [comment]
(1) Line 8 shouldn’t mention macOS.

(2) On macOS, there’s an issue with System Integrity Protection: the `alr` binary is quarantined,
```
$ xattr -v bin/alr 
bin/alr: com.apple.quarantine
```
If you try to run it on recent versions of macOS, you’ll get a popup saying '“alr” cannot be opened because the developer cannot be verified.' and inviting you to move it to the bin.

The way round this is to remove the quarantine attribute,
```
$ xattr -d com.apple.quarantine bin/alr
```

File: doc/getting-started.md [hunk 1]
```diff
@@ -28,6 +30,22 @@ otherwise have to install manually. `msys2` will also provide external
...
+On macOS, `Alire` is simply provided in an archive.
+
+Once the archive is extracted you have to add `alr` in the environment `PATH`:
+```bash
+$ export PATH=<PATH_TO_EXTRACTED>/bin/:$PATH
+```
+
+Alire provides GNAT toolchains hosted on x86-64 for macOS. If those toolchains do not 
+work for you, or if you are on another host architecture like the Apple M1, you have
+the option look at the GNAT toolchains from the community.
```
Comment (mosteo) [inline doc/getting-started.md]: option [to] look

**Review by mosteo — CHANGES_REQUESTED** [review]
Just a typo.

Also adding the info provided by Simon looks useful for macOS.

**Comment by Fabien-Chouteau** [comment]
Thanks both :+1:

**Review by mosteo — APPROVED** [review]

**Pull request merged**